### PR TITLE
fix(dms/group): deprecate huaweicloud_dms_group

### LINKED ISSUE #1876: [DMS] Distributed Message Service (Shared Edition) has been withdrawn, dreprecate the resources of DMS
Distributed Message Service (Shared Edition) has been withdrawn, dreprecate the resources of DMS, 
the follows:
- huaweicloud_dms_queue
- huaweicloud_dms_group

### PR DESCRIPTION
**Pull request by chengxiangdong**
**What this PR does / why we need it**:

Distributed Message Service (Shared Edition) has been withdrawn, we need to dreprecate the resource: `huaweicloud_dms_group`.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #1876

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```

```

### CODE REVIEW

File: docs/resources/dms_group.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ---
-subcategory: "Distributed Message Service (DMS)"
+subcategory: "Deprecated"
 ---
 
 # huaweicloud_dms_group
 
 Manages a DMS group in the huaweicloud DMS Service.
 
+-> **NOTE:** Distributed Message Service (Shared Edition) has withdrawn. Please use DMS for Kafka instead.
```
Comment (ShiChangkuo) [inline docs/resources/dms_group.md:9]: suggest to use the exact resource name.

File: huaweicloud/provider.go [hunk 2]
```diff
@@ -482,7 +482,7 @@ func Provider() *schema.Provider {
 			"huaweicloud_dli_table":                       dli.ResourceDliTable(),
 			"huaweicloud_dli_flinksql_job":                dli.ResourceFlinkSqlJob(),
 			"huaweicloud_dli_flinkjar_job":                dli.ResourceFlinkJarJob(),
-			"huaweicloud_dms_group":                       dms.ResourceDmsGroups(),
+			"huaweicloud_dms_group":                       deprecated.ResourceDmsGroups(),
```
Comment (ShiChangkuo) [inline huaweicloud/provider.go]: suggest move it to the **Deprecated** session which at the last of map.

**Comment by niuzhenguo** [comment]
Does this resource still work? if it is, pls paste the test results according to the PR checklist.

**Comment by chengxiangdong** [comment]
> Does this resource still work? if it is, pls paste the test results according to the PR checklist.

unable to create new resources, and the unit tests don't work

File: docs/resources/dms_group.md (hunk 1 above)
Comment (chengxiangdong) [reply docs/resources/dms_group.md:9]: Kafka does not have the features of group and queue, so it is impossible to specify exact resource name

File: huaweicloud/provider.go (hunk 2 above)
Comment (chengxiangdong) [reply huaweicloud/provider.go]: done

**Pull request merged**